src/main.cpp

```diff
@@ -837,6 +850,7 @@ QString UsageArguments ( char **argv )
         "  -m, --htmlstatus      enable HTML status file, set file name\n"
         "  -o, --serverinfo      infos of this server in the format:\n"
         "                        [name];[city];[country as QLocale ID]\n"
+        "  -P, --delayedpan      start with delayed panning enabled\n"
```

Comment:
```suggestion
        "  -P, --delayedpanning  start with delayed panning enabled\n"
```

Reply:
This should be called "delaypan" or "delaypanning". I don't know where the term "delayed panning" comes from, but it is not accurate, because the panning is not delayed. The panning uses delays to achieve its function. Might be pedantic, but it shouldn't give the impression that this effect is delayed (and increases latency) in any way.

Reply:
Would be fine for me, I just suggested the change because the help text currently does not match the implementation. It has to be changed either wa. :)

Reply:
The English term is 'delay panning' and the short form 'pan' is often used instead of 'panning'. So both 'delay panning' or  'delay pan' would be OK.

Reply:
@hoffie where do you see this doesn’t match the implementation?

Reply:
The help text says `--delayedpan` (the line I commented on) and the implementation parses `--delayedpanning` ([line 389](https://github.com/jamulussoftware/jamulus/pull/1151/files#diff-34d21af3c614ea3cee120df276c9c4ae95053830d7f1d3deaf009a4625409ad2R389))

Reply:
Ok, I've just fixed that and use delaypan/delay panning throughout.

I am still wondering, though, if we shouldn't make this default to on. As Detlef Hennings wrote essentially nobody would go back to volume panning once they heard delay panning and I very much agree. Especially choirs will benefit from this, so this will be a lot of people, and from my experience with helping them they struggle a lot with setting up their own server. I think most won't touch anything again as they are glad the thing works at all. They would all miss out on this if we ask them to set another option. On the other hand, those who will be annoyed by the new sound will figure out how to go back if we provide the option for it.

Reply:
Could you please rename the option to either '--delaypan' or '--delaypanning', but please do not use '--delayedpan' and '--delayedpannning', which are no correct terms.   

My suggestion to make delpan public is to give the information, if delay or volume panning is used, in the public server lists. This could be done by adding a (d) for delay panning and a (v) for volume panning behind the musicians count/capacity, e.g.  18/30 (d)  or  5/8 (v).

Reply:
`--delaypan` is what I set it to.

I was rather thinking of private server installations, which is what a lot of choirs use. If we make the default _on_ people would just have to update their server and get it. If we also have to make them update their service installation they will be lost.

Announcement in the server list is beyond of what I did here.

Reply:
--delaypan  is fine (I didn't see, that it was already changed). 

Delay pan as default, that's what I would also prefer. Maybe to some users it will be a (positive) surprise then ...